x-pack/plugins/security_solution/public/common/containers/source/index.tsx

```diff
@@ -158,15 +158,20 @@ export const useFetchIndex = (
             next: (response) => {
               if (isCompleteResponse(response)) {
                 const stringifyIndices = response.indicesExist.sort().join();
+
                 previousIndexesName.current = response.indicesExist;
-                setLoading(false);
+                setLoading((prevValue) => {
```

Comment:
what do we gain here?

Reply:
I don't know, it is the same right? I rollback this line